### DISCUSSION TITLE
(gatsby-source-strapi) sort multiple fields

### DISCUSSION THREAD

**Discussion by yakou32** [gatsby-uc/plugins#390] [Q&A]
Hello!
I don't understand so well how to sort from multiple fields in graphQL.

This is the syntax suggested by GraphiQL: `sort: {order: ASC, name: ASC}`

But this generates an `Invalid field arg` error...

Would you have an idea or some documentation about sorting on multiple fields?

**Comment by moonmeister** [comment]
Checkout the Gatsby docs for how they've specifically implemented sorting: https://www.gatsbyjs.com/docs/graphql-reference/#sort
